### PR TITLE
Display upcoming notice in list publications

### PR DESCRIPTION
**Pull request by jrlarano**

### CODE REVIEW

File: locales/nb_NO.ts [hunk 1]
```diff
@@ -20,5 +20,6 @@ export default {
     publication_viewer_overview_button: 'oversikt',
     publication_viewer_close_label: 'Tilbake',
     publication_viewer_add_to_shopping_list: 'Legg til handleliste',
-    publication_viewer_visit_webshop_link: 'Besøk nettbutikklink'
+    publication_viewer_visit_webshop_link: 'Besøk nettbutikklink',
+    publication_viewer_upcoming: 'Påkommende avis'
```
Comment (mortenbo) [inline locales/nb_NO.ts]: Remove "avis"

File: locales/sv_SE.ts [hunk 2]
```diff
@@ -19,5 +19,6 @@ export default {
     publication_viewer_overview_button: 'Översikt',
     publication_viewer_close_label: 'Tillbaka',
     publication_viewer_add_to_shopping_list: 'Lägg till inköpslista',
-    publication_viewer_visit_webshop_link: 'Besök webbshoplänk'
+    publication_viewer_visit_webshop_link: 'Besök webbshoplänk',
+    publication_viewer_upcoming: 'Kommende tidning'
```
Comment (mortenbo) [inline locales/sv_SE.ts]: Remove "tidning"

**Comment by mortenbo** [comment]
@jrlarano Cool, how does this look?

**Comment by jrlarano** [comment]
> @jrlarano Cool, how does this look?

@mortenbo here's how it looks 🙂 
image (14)

**Comment by mortenbo** [comment]
Can you do it in this style instead so it's streamlined and more visible? I didn't see the white version 😅 And maybe make the upcoming pub a bit faded like in the screenshot too.

Screenshot 2023-08-14 at 10 33 56

**Comment by jrlarano** [comment]
@mortenbo Something like this? 🙂 

image

**Comment by mortenbo** [comment]
Yeah but perhaps align the label to the bottom of the image.

**Comment by jrlarano** [comment]
@mortenbo Like this? 🙂 

image

**Comment by mortenbo** [comment]
Great!

**Review by klarstrup — APPROVED** [review]
LGTM!

**Comment by mortenbo** [comment]
Cool! Did we release it?

**Comment by jrlarano** [comment]
Not yet.

 @klarstrup can you help us in releasing this? Thank you! 🙂

**Comment by klarstrup** [comment]
4.16.0 is out now 🙂

https://github.com/tjek/tjek-js-sdk/releases/tag/release-17.08.2023-12.20

**Pull request merged**